### DISCUSSION TITLE
After verifying in the web page and returning to pycharm it keeps on ‘waiting for Github Authentication

### DISCUSSION THREAD

**Discussion by jonlai211** [community/community#16903] [Copilot Conversations]
After verifying in the web page and returning to pycharm it keeps on ‘waiting for Github Authentication lol
![image](https://user-images.githubusercontent.com/84857377/169244323-873f3a94-99aa-46d0-9fbd-ee7f69e4ae9e.png

**Comment by jansorg** [comment]
@Jonathan-Lai0101 Are you perhaps using a proxy server?

**Reply by jonlai211** [reply]
with a VPN at that time, but it can work successfully before. So weird....
BTW after change my Pycharm to the newest version,  it works out.

**Reply by tryinggo** [reply]
What version of pycharm do you have?

**Reply by jonlai211** [reply]
past 2021.3 ❌
now 2022.1 ✔
